lib/inspec/schema/exec_json.rb

```diff
@@ -0,0 +1,73 @@
+require_relative "./primitives"
+
+# We disable this for the sake of legibility
+# rubocop:disable Layout/AlignHash
+
+# These type occur only when running "inspec exec --reporter json <file>".
+
+module Schema
+  module ExecJson
+    # Represents a control produced
+    CONTROL = SchemaType.new("Exec JSON Control", {
+      "additionalProperties" => false,
+      "required" => %w{id title desc descriptions impact refs tags code source_location results},
+      "properties" => {
+        "id"                => STRING,
+        "title"             => { "type" => %w{string null} }, # Nullable string
+        "desc"              => { "type" => %w{string null} },
+        "descriptions"      => Schema.array(CONTROL_DESCRIPTION.ref),
+        "impact"            => IMPACT,
+        "refs"              => Schema.array(REFERENCE.ref),
+        "tags"              => TAGS,
+        "code"              => STRING,
+        "source_location"   => SOURCE_LOCATION.ref,
+        "results"           => Schema.array(CONTROL_RESULT.ref),
+      },
+    }, [CONTROL_DESCRIPTION, REFERENCE, SOURCE_LOCATION, CONTROL_RESULT])
+
+    # Based loosely on https://www.inspec.io/docs/reference/profiles/ as of July 3, 2019
+    # However, concessions were made to the reality of current reporters, specifically
+    # with how description is omitted and version/inspec_version aren't as advertised online
+    PROFILE = SchemaType.new("Exec JSON Profile", {
```

Comment:
even if version/inspec_version aren't as advertised we should still include them.

Reply:
Done